src/style.css

```diff
@@ -0,0 +1,18 @@
+main {
+  margin: 0 auto;
+  background-color: white;
+  border-top: 20px black solid;
+  border-bottom: 20px black solid;
```

Comment:
- [ ] You don't need these borders, check the original design:

![image](https://user-images.githubusercontent.com/60631456/126229114-23fa69b7-0193-4d37-9623-5d1089f5a599.png)

Reply:
* done 👍